web/src/components/RunBottomPanel/Header.tsx

```diff
@@ -61,7 +68,25 @@ const Header: React.FC<IProps> = ({run: {createdAt}, assertionResults, isDisable
           <Badge count="F" /> <S.CountNumber>{totalFailedCount}</S.CountNumber>
         </S.HeaderText>
       </div>
-      <div>
+      <S.Row>
+        <Tooltip
+          color="#FBFBFF"
+          title={`
+            You can decided wether you want to see the results using the key-value (wizard) mode or the query language. 
```

Comment:
typo: `You can decide whether you want to see the results using the key-value (wizard) mode or the query language (advanced).`